src/test/java/com/exasol/release/GitHubReleaseMakerTest.java

```diff
@@ -0,0 +1,23 @@
+package com.exasol.release;
+
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.mockito.Mockito.*;
+
+import org.junit.jupiter.api.Test;
+
+import com.exasol.github.GitHubRepository;
+
+class GitHubReleaseMakerTest {
+    @Test
+    void testMakeRelease() {
+        final String version = "1.0.0";
+        final GitHubRepository gitHubRepositoryMock = mock(GitHubRepository.class);
+        when(gitHubRepositoryMock.getVersion()).thenReturn(version);
+        when(gitHubRepositoryMock.getChangesFile(version)).thenReturn("Release \n letter");
```

Comment:
Add tests where there is no newline or no content after the newline. Both cases should throw errors.

Reply:
done